src/OLEDDisplay.h

```diff
@@ -222,6 +222,9 @@ class OLEDDisplay : public Print {
     // normal brightness & contrast:  contrast = 100
     void setContrast(uint8_t contrast, uint8_t precharge = 241, uint8_t comdetect = 64);
 
+    // Set clock divider (lower 4 bits) and oscillator frequency (higher 4 bits)
```

Comment:
It would be helpful if you also documented for users _why_ one should want to change this (i.e. describe use cases).